### PR TITLE
Feature/aditional support page

### PR DESCRIPTION
**Pull request by andrejskalnins**
I noticed that we are using react router v5.4 not the latest one v6 
Side note: I'll clean up the class names to more readable ones.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -26,7 +27,15 @@ function App() {
 					<Route path="/about-us-more" component={AboutUsMore} />
 					<Route path="/whatwedo" component={WhatWeDoMore} />
 					{/* routes from nav bar: */}
-					<Route path="/about" component={About} />
+
```
Comment (youssef910) [inline client/src/App.js:30]: check the indentation

File: client/src/pages/AdditionalSupport/AdditionalSupport.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	line-height: 21px;
+}
+.content-child1 p:nth-child(2) a{
+	font-family: "Ambit Regular";
+}
+
+.content-child1 > p:nth-child(2) {
+	width: 315px;
+	font-family: "Ambit Light";
+
```
Comment (youssef910) [inline client/src/pages/AdditionalSupport/AdditionalSupport.css:37]: remove empty lines

File: client/src/pages/AdditionalSupport/AdditionalSupport.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	return (
+		<div className="additional-support-container">
+			<div><span></span></div>
+			<img
+				className="hero-image"
+				src={additionalSupportImg}
+				alt="sensational life logo"
+			></img>
+			<div className="content">
+				<div className="content-child1">
```
Comment (youssef910) [inline client/src/pages/AdditionalSupport/AdditionalSupport.js:18]: indentation again

**Review by youssef910 — CHANGES_REQUESTED** [review]

File: client/src/pages/AdditionalSupport/AdditionalSupport.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+function AdditionalSupport() {
+	return (
+		<div className="additional-support-container">
+			<div><span></span></div>
+			<img
+				className="hero-image"
+				src={additionalSupportImg}
+				alt="sensational life logo"
+			></img>
```
Comment (khaledkz) [inline client/src/pages/AdditionalSupport/AdditionalSupport.js:16]: you should use self close tag for

**Review by khaledkz — APPROVED** [review]
looks good, just left a minor comments also need to fix indentation

**Review by youssef910 — APPROVED** [review]

**Pull request merged**